controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -473,6 +478,15 @@ func getFailureDomainKeyFromStorageClassParameter(sc *storagev1.StorageClass) st
 	}
 }
 
+func (r *OCSInitializationReconciler) allowRemoteStorageConsumers() bool {
+	for _, sc := range r.clusters.GetInternalStorageClusters() {
+		if sc.Spec.AllowRemoteStorageConsumers {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
I am not sure why we need to implement this function at all

Reply:
done. used `slices.ContainsFunc`

Reply:
Nice knowing that Go finally added a useful slice util func